news/2023/2023-09-09-monthly-beatmapping-contest-september-2023.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Monthly Beatmapping Contest: September 2023"
+date: 2023-09-09 18:00:00 +0000
+---
+
+This classic contest makes a comeback yet again! See if you have what it takes to take home the gold and make your way towards the coveted Elite Mapper title.
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+Let's be honest: the [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) (*MBC*) isn't anything special — but that may be what makes it so valuable.
+
+This is a plain 'n' simple mapping contest, allowing mappers to show off their talents! (...and earn some [prizes](#prizes) along the way.)
+
+Let's break it down:
+
+## Game modes
+
+This month's contest is open to **osu!** and **osu!mania** mappers.
+
+- [osu! contest listing](https://osu.ppy.sh/community/contests/184)
+- [osu!mania contest listing](https://osu.ppy.sh/community/contests/185)
+
+Next month's contest will feature **osu!taiko** and **osu!catch**.
+
+## Timeline
+
+- [**Beatmapping (3 weeks):**](#beatmapping) Contest is announced and beatmappers create their entries.
+- [**Screening (1.5 weeks):**](/wiki/Contests/Monthly_Beatmapping_Contest#screening) Entries are evaluated by experienced beatmappers. If there aren't many entries, this will be skipped in favour of a longer judging period.
+- [**Judging (1.5 weeks):**](/wiki/Contests/Monthly_Beatmapping_Contest#judging) Top entries from screening are judged in detail by a different crew of experienced beatmappers.
+- [**Rewards (~2 months from now):**](#rewards) Winners are announced at the start of another MBC.
+
+## Beatmapping
+
+### Limitation
+
+Creating a quality beatmap is a solid goal, but true creativity comes when your tools are restricted.
+
+That's why we're adding **limitations** to this contest!
+
+#### osu! limitation
+
+This month's osu! limitation: **all sliders must be curved.**
+
+More specifically, *every segment of a slider must be curved.* Here's some visual examples to clear things up:
+
+![](/wiki/shared/news/2023-09-04-monthly-beatmapping-contest-september-2023/limitexample.png)
+
+Part of the judging criteria involves how you handle the limitation, so keep that in mind if you're creating [sliders that are *technically curved* yet look exactly like straight sliders](/wiki/shared/news/2023-09-04-monthly-beatmapping-contest-september-2023/technicality.png).
```

Comment:
so... are these types of sliders allowed or not?

Reply:
allowed but not a good idea to use frequently if mappers want to score highly in a certain judging category